File: FIGLet.VSCodeExtension/src/FIGLet/LayoutMode.ts
/*
 *   _                       _   __  __         _     
 *  | |   __ _ _  _ ___ _  _| |_|  \/  |___  __| |___ 
 *  | |__/ _` | || / _ \ || |  _| |\/| / _ \/ _` / -_)
 *  |____\__,_|\_, \___/\_,_|\__|_|  |_\___/\__,_\___|
 *             |__/                                   
 */
/**
 * Specifies the layout mode for rendering FIGLet text.
 */
export enum LayoutMode {
    /**
     * Full size layout mode.
     */
    FullSize = -1,

    /**
     * Kerning layout mode.
     */
    Kerning = 0,

    /**
     * Smushing layout mode.
     */
    Smushing = 1,

    /**
     * The default layout mode, which is smushing.
     */
    Default = Smushing
}

File: FIGLet/LayoutMode.cs
namespace ByteForge.FIGLet;

/*
 *  _                       _   __  __         _     
 * | |   __ _ _  _ ___ _  _| |_|  \/  |___  __| |___ 
 * | |__/ _` | || / _ \ || |  _| |\/| / _ \/ _` / -_)
 * |____\__,_|\_, \___/\_,_|\__|_|  |_\___/\__,_\___|
 *            |__/                                   
 */

/// <summary>
/// Specifies the layout mode for rendering FIGLet text.
/// </summary>
public enum LayoutMode
{
    /// <summary>
    /// Full size layout mode.
    /// </summary>
    FullSize = -1,

    /// <summary>
    /// Kerning layout mode.
    /// </summary>
    Kerning = 0,

    /// <summary>
    /// Smushing layout mode.
    /// </summary>
    Smushing = 1,

    /// <summary>
    /// The default layout mode, which is smushing.
    /// </summary>
    Default = Smushing
}